Failing AF Uproot Tests

```
json_query = {'chunk-size': '1000', 'file-list': ['root://eospublic.cern.ch//eos/opendata/atlas/OutreachDatasets/2020-01-22/4lep/MC/mc_345060.ggH125_ZZ4lep.4lep.root'], 'result-destination': 'object-store', 'result-format': 'root-file', ...}

    async def submit_query(self, client: aiohttp.ClientSession,
                           json_query: Dict[str, str]) -> Dict[str, str]:
        """
        Submit a query to ServiceX, and return a request ID
        """
    
        headers = await self._get_authorization(client)
    
        async with client.post(f'{self._endpoint}/servicex/transformation',
                               headers=headers, json=json_query) as response:
            status = response.status
            if status != 200:
                # This was an error at ServiceX, bubble it up so code above us can
                # handle as needed.
                t = await response.text()
>               raise ServiceXException('ServiceX rejected the transformation request: '
                                        f'({status}){t}')
E               servicex.utils.ServiceXException: (ServiceXException(...), 'ServiceX rejected the transformation request: (400){"message": "Failed to submit transform request: Failed to generate translation code: Unknown id: ResultTTree"}\n')

/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/servicex/servicex_adaptor.py:66: ServiceXException
=============================== warnings summary ===============================
../../../../../opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pytest_asyncio/plugin.py:1[91](https://github.com/ssl-hep/servicex-backend-tests/runs/5253074614?check_suite_focus=true#step:5:91)
  /opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/pytest_asyncio/plugin.py:191: DeprecationWarning: The 'asyncio_mode' default value will change to 'strict' in future, please explicitly use 'asyncio_mode=strict' or 'asyncio_mode=auto' in pytest configuration file.
    config.issue_config_time_warning(LEGACY_MODE, stacklevel=2)
```

--- a/.github/workflows/daily_servicex_uproot_test_af.yml
+++ b/.github/workflows/daily_servicex_uproot_test_af.yml
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7]
+        python-version: [3.9]
 
     steps:
       - uses: actions/checkout@v2

--- a/.github/workflows/daily_servicex_uproot_test_af.yml
+++ b/.github/workflows/daily_servicex_uproot_test_af.yml
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: [3.7]
 
     steps:
       - uses: actions/checkout@v2

--- a/.github/workflows/daily_servicex_uproot_test_af.yml
+++ b/.github/workflows/daily_servicex_uproot_test_af.yml
@@ -26,6 +26,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install flake8 pytest
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          pip list
       - name: Execute Uproot tests
         env:
           TOKEN: ${{ secrets.SECRET_AF_UPROOT_TOKEN }}

--- a/.gitignore
+++ b/.gitignore
@@ -5,3 +5,4 @@ __pycache__/
 kube/ign*
 .idea/
 river-dashboard-values.yaml
+servicex.yaml

--- a/test_uproot.sh
+++ b/test_uproot.sh
@@ -1,6 +1,6 @@
 # A simple shell script that will generate a servicex.yaml file and automatically run it, primarily meant for GitHub actions.
 # David Liu, 21 September 2020, Seattle, WA.
 
-printf "api_endpoints:\n  - endpoint: $2\n    token: $1\n    name: uproot" > ./servicex.yaml
+printf "api_endpoints:\n  - endpoint: $2\n    token: $1\n    name: uproot\n    type: uproot" > ./servicex.yaml
 pytest -s tests/test_uproot_functions.py
 rm ./servicex.yaml

